tty/zynq7000-uart/zynq7000-uart.c

```diff
@@ -455,45 +416,58 @@ int main(int argc, char **argv)
 	uart_t *uart;
 	speed_t baud = B115200;
 	uint32_t port;
-	int n, c, console = 1;
-
-	/* Get console ID and baudrate */
-	while ((c = getopt(argc, argv, "b:c:")) != -1) {
-		switch (c) {
-			case 'b':
-				if ((baud = libtty_int_to_baudrate(atoi(optarg))) == (speed_t)-1) {
-					debug("uart: wrong baudrate value\n");
-					return EXIT_FAILURE;
-				}
-				break;
-			case 'c':
-				console = atoi(optarg);
-				if (console < 0 || console >= UARTS_MAX_CNT) {
-					debug("uart: wrong console ID\n");
-					return EXIT_FAILURE;
-				}
-				break;
-			default:
-				break;
+	int c;
+	unsigned int uartn = -1;
```

Comment:
You set it to 1 if argc == 1 anyway, no need for this

```suggestion
	unsigned int uartn = 1;
```